Altis_Life.Altis/core/actions/fn_repairTruck.sqf

```diff
@@ -49,22 +50,23 @@ if ((_veh isKindOf "Car") || (_veh isKindOf "Ship") || (_veh isKindOf "Air")) th
         if (life_interrupted) exitWith {life_interrupted = false; titleText[localize "STR_NOTF_ActionCancel","PLAIN"]; life_action_inUse = false;};
         if !(isNull objectParent player) exitWith {titleText[localize "STR_NOTF_ActionInVehicle","PLAIN"];};
 
-        _sideRepairArray = LIFE_SETTINGS(getArray,"vehicle_infiniteRepair");
+        private _sideRepairArray = LIFE_SETTINGS(getArray,"vehicle_infiniteRepair");
 
         //Check if playerSide has infinite repair enabled
-        if (playerSide isEqualTo civilian && (_sideRepairArray select 0) isEqualTo 0) then {
-            [false,"toolkit",1] call life_fnc_handleInv;
-        };
-        if (playerSide isEqualTo west && (_sideRepairArray select 1) isEqualTo 0) then {
-            [false,"toolkit",1] call life_fnc_handleInv;
-        };
-        if (playerSide isEqualTo independent && (_sideRepairArray select 2) isEqualTo 0) then {
-            [false,"toolkit",1] call life_fnc_handleInv;
-        };
-        if (playerSide isEqualTo east && (_sideRepairArray select 3) isEqualTo 0) then {
-            [false,"toolkit",1] call life_fnc_handleInv;
+        call {
```

Comment:
The index needs to match up with the side, which would just become a similar idea to this already. However, I think the idea here was to use `exitWith`, so it doesn't keep evaluating. call {} & exitWith helps a lot with logic (to avoid a nasty if w/ lazy eval, or if elses